test/resources/issue-101.js

```diff
@@ -0,0 +1,112 @@
+'use strict'
+
+function names(arg) {
```

Comment:
For all the methods (why is not failing the linter?)
```suggestion
function names (arg) {
```

Reply:
`test/resources` is [ignored](https://github.com/DataDog/dd-native-iast-rewriter-js/blob/main/.eslintignore#L2)  

But i'm going to rename the methods

Reply:
done